src/app/public/modules/progress-indicator/progress-indicator-item/progress-indicator-item.component.html

```diff
@@ -1,24 +1,28 @@
-<div *ngIf="!isHorizontal || isActive"
-  class="sky-progress-indicator-item-heading"
-  [ngClass]="headingClassList"
+<div *ngIf="isVisible"
+  class="sky-progress-indicator-item"
+  [ngClass]="'sky-progress-indicator-item-status-' + statusName"
 >
-  <div
-    class="sky-progress-indicator-item-title"
-    [ngClass]="{
-      'sky-emphasized': isHorizontal,
-      'sky-deemphasized': isPassive && isActive
-    }"
+  <sky-progress-indicator-status-marker *ngIf="showStatusMarker"
+    [status]="status"
   >
-    {{ displayTitle }}
-  </div>
-</div>
-<div *ngIf="isLineDisplayed"
-  class="sky-progress-indicator-item-line"
-  [ngClass]="lineClassList"
->
-  <div *ngIf="isActive || isComplete"
-    class="sky-progress-indicator-item-content"
+  </sky-progress-indicator-status-marker>
+
+  <div
+    class="sky-progress-indicator-item-main"
   >
-    <ng-content></ng-content>
+    <div *ngIf="showTitle"
+      class="sky-progress-indicator-item-title"
+      role="heading"
+      [ngClass]="{ 'sky-deemphasized': statusName === 'pending' }"
+    >
+      {{ titlePrefix }}{{ title }}
```

Comment:
Good call.

Reply:
Does having these two variables back-to-back in the template cause any concern? Spacing is important between these two variables and if an unknowing dev changed this in the future, it could inadvertently add or remove spaces. Would it be better to control this with a single display variable in the component logic?